head/src/main/kotlin/io/qalipsis/core/head/campaign/PersistentCampaignService.kt

```diff
@@ -125,26 +125,37 @@ internal class PersistentCampaignService(
         tenant: String,
         campaignKey: CampaignKey,
         result: ExecutionStatus,
-        failureReason: String?
+        failureReason: String?,
     ): Campaign {
         campaignRepository.complete(tenant, campaignKey, result, failureReason)
         return retrieve(tenant, campaignKey)
     }
 
     @LogInput
     override suspend fun search(
-        tenant: String, filters: Collection<String>, sort: String?, page: Int, size: Int
+        tenant: String,
+        filters: Collection<String>,
+        sort: String?,
+        page: Int,
+        size: Int,
+        excludedStatuses: Collection<ExecutionStatus>,
     ): QalipsisPage<Campaign> {
         // Default sorting for the campaign is done with the start time in reverse order, because it is always not null.
         val sorting = sort?.let { SortingUtil.sort(CampaignEntity::class, it) }
             ?: Sort.of(Sort.Order.desc(CampaignEntity::start.name))
-
         val pageable = Pageable.from(page, size, sorting)
 
         val entitiesPage = if (filters.isNotEmpty()) {
-            campaignRepository.findAll(tenant, filters.formatsFilters(), pageable)
+            campaignRepository.findAll(
```

Comment:
Please add the argument names, because the statuses are excluded while the other values are included. This is misleading.